### ISSUE TITLE
JSON color scheme not applied when "the" is written on example

### ISSUE THREAD

**Issue by ghost** [getinsomnia/insomnia#4361]
### Expected Behavior

The previsualised JSON should be colored as shown (Edited Image): 
![image](https://user-images.githubusercontent.com/73848304/149325005-9c05b816-de6d-48a4-9b7c-1a5987e609fa.png)

### Actual Behavior

The previsualised JSON currently don't have any colors as shown : 
![image](https://user-images.githubusercontent.com/73848304/149325190-e9acbcaf-04d4-4c11-909a-820babd533ea.png)

### Reproduction Steps

- Create a new document
- Create a path
- Inside the responses add a new response
- Inside this response, fill the content with JSON object 
- Inside this object, create three properties, add an example with "the" inside
- **You should see colors on the preview**
- Then add a fourth property with "the" in the example
- **You should see that the color in the preview has disappeared** 

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

For easy reproduction, you can use [this](https://haste.daniels-roth-stan.fr/iveluzadub.json) JSON
I use a custom theme that comes with a plugin ([insomnia-plugin-theme-tokyo-night](https://insomnia.rest/plugins/insomnia-plugin-theme-tokyo-night)), but i've tried with and without and the problem still occurs.  

### Insomnia Version

2021.7.2

### What operating system are you using?

Windows

### Operating System Version

Windows 11 Family Insider Preview 22526.1000

### Installation method

download from insomnia.rest

**Comment by MrStanDu33** [comment]
Hello there,

Tried on the same Insomnia version as you, on the same Windows version, and it is, in fact, reproductible.

**Comment by MrStanDu33** [comment]
I also can reproduce the issue by replacing **"the"** with **"an"** and **"a"** 🤔

**Comment by MrStanDu33** [comment]
A little update here 👋,

I found out that the Issue come from the [`SyntaxHighlighter`](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/src/index.js) component of the [`react-syntax-highlighter`](https://github.com/react-syntax-highlighter/react-syntax-highlighter) lib. But I can't figure out why, as props seems to be the same between the two cases (except the missing "the") 🔎.

You can compare props here :

❌ Broken color scheme props 
```json
{
  "className": "example microlight",
  "onWheel": "ƒ () {}",
  "style": {
    "hljs": "{background: \"#333\", color: \"white\", display: \"bloc…}",
    "hljs-name": "{fontWeight: \"bold\"}",
    "hljs-strong": "{fontWeight: \"bold\"}",
    "hljs-code": "{color: \"#888\", fontStyle: \"italic\"}",
    "hljs-emphasis": "{fontStyle: \"italic\"}",
    "hljs-tag": "{color: \"#62c8f3\"}",
    "hljs-variable": "{color: \"#ade5fc\"}",
    "hljs-template-variable": "{color: \"#ade5fc\"}",
    "hljs-selector-id": "{color: \"#ade5fc\"}",
    "hljs-selector-class": "{color: \"#ade5fc\"}",
    "hljs-string": "{color: \"#a2fca2\"}",
    "hljs-bullet": "{color: \"#d36363\"}",
    "hljs-type": "{color: \"#ffa\"}",
    "hljs-title": "{color: \"#ffa\"}",
    "hljs-section": "{color: \"#ffa\"}",
    "hljs-attribute": "{color: \"#ffa\"}",
    "hljs-quote": "{color: \"#ffa\"}",
    "hljs-built_in": "{color: \"#ffa\"}",
    "hljs-builtin-name": "{color: \"#ffa\"}",
    "hljs-number": "{color: \"#d36363\"}",
    "hljs-symbol": "{color: \"#d36363\"}",
    "hljs-keyword": "{color: \"#fcc28c\"}",
    "hljs-selector-tag": "{color: \"#fcc28c\"}",
    "hljs-literal": "{color: \"#fcc28c\"}",
    "hljs-comment": "{color: \"#888\"}",
    "hljs-deletion": "{backgroundColor: \"#fc9b9b\", color: \"#333\"}",
    "hljs-regexp": "{color: \"#c6b4f0\"}",
    "hljs-link": "{color: \"#c6b4f0\"}",
    "hljs-meta": "{color: \"#fc9b9b\"}",
    "hljs-addition": "{backgroundColor: \"#a2fca2\", color: \"#333\"}"
  },
  "children": "{\n  \"errors\": {\n    \"name\": \"Name doesn't have the correct format\",\n    \"firstname\": \"Firstname doesn't have the correct format\",\n    \"username\": \"Username doesn't have the correct format\",\n    \"email\": \"Username doesn't have correct the format\",\n    \"password\": \"Password doesn't have ee correct format\",\n    \"question\": \"Question doesn't have  correct format\",\n    \"reponse\": \"Reponse doesn't have  correct format\"\n  }\n}"
}
```

✅ Normal color scheme props
```json
{
  "className": "example microlight",
  "onWheel": "ƒ () {}",
  "style": {
    "hljs": "{background: \"#333\", color: \"white\", display: \"bloc…}",
    "hljs-name": "{fontWeight: \"bold\"}",
    "hljs-strong": "{fontWeight: \"bold\"}",
    "hljs-code": "{color: \"#888\", fontStyle: \"italic\"}",
    "hljs-emphasis": "{fontStyle: \"italic\"}",
    "hljs-tag": "{color: \"#62c8f3\"}",
    "hljs-variable": "{color: \"#ade5fc\"}",
    "hljs-template-variable": "{color: \"#ade5fc\"}",
    "hljs-selector-id": "{color: \"#ade5fc\"}",
    "hljs-selector-class": "{color: \"#ade5fc\"}",
    "hljs-string": "{color: \"#a2fca2\"}",
    "hljs-bullet": "{color: \"#d36363\"}",
    "hljs-type": "{color: \"#ffa\"}",
    "hljs-title": "{color: \"#ffa\"}",
    "hljs-section": "{color: \"#ffa\"}",
    "hljs-attribute": "{color: \"#ffa\"}",
    "hljs-quote": "{color: \"#ffa\"}",
    "hljs-built_in": "{color: \"#ffa\"}",
    "hljs-builtin-name": "{color: \"#ffa\"}",
    "hljs-number": "{color: \"#d36363\"}",
    "hljs-symbol": "{color: \"#d36363\"}",
    "hljs-keyword": "{color: \"#fcc28c\"}",
    "hljs-selector-tag": "{color: \"#fcc28c\"}",
    "hljs-literal": "{color: \"#fcc28c\"}",
    "hljs-comment": "{color: \"#888\"}",
    "hljs-deletion": "{backgroundColor: \"#fc9b9b\", color: \"#333\"}",
    "hljs-regexp": "{color: \"#c6b4f0\"}",
    "hljs-link": "{color: \"#c6b4f0\"}",
    "hljs-meta": "{color: \"#fc9b9b\"}",
    "hljs-addition": "{backgroundColor: \"#a2fca2\", color: \"#333\"}"
  },
  "children": "{\n  \"errors\": {\n    \"name\": \"Name doesn't have the correct format\",\n    \"firstname\": \"Firstname doesn't have the correct format\",\n    \"username\": \"Username doesn't have the correct format\",\n    \"email\": \"Username doesn't have correct  format\",\n    \"password\": \"Password doesn't have ee correct format\",\n    \"question\": \"Question doesn't have  correct format\",\n    \"reponse\": \"Reponse doesn't have  correct format\"\n  }\n}"
}
```

But then, the HTML generated by the `SyntaxHighlighter` looks like this :

❌ Broken color scheme props
```html
<code>{
  <span style="color: rgb(136, 136, 136);">"errors"</span>: {
    <span style="color: rgb(136, 136, 136);">"name"</span>: <span style="color: rgb(136, 136, 136);">"Name doesn't have the correct format"</span>,
    <span style="color: rgb(136, 136, 136);">"firstname"</span>: <span style="color: rgb(136, 136, 136);">"Firstname doesn't have the correct format"</span>,
    <span style="color: rgb(136, 136, 136);">"username"</span>: <span style="color: rgb(136, 136, 136);">"Username doesn't have the correct format"</span>,
    <span style="color: rgb(136, 136, 136);">"email"</span>: <span style="color: rgb(136, 136, 136);">"Username doesn't have correct the format"</span>,
    <span style="color: rgb(136, 136, 136);">"password"</span>: <span style="color: rgb(136, 136, 136);">"Password doesn't have ee correct format"</span>,
    <span style="color: rgb(136, 136, 136);">"question"</span>: <span style="color: rgb(136, 136, 136);">"Question doesn't have  correct format"</span>,
    <span style="color: rgb(136, 136, 136);">"reponse"</span>: <span style="color: rgb(136, 136, 136);">"Reponse doesn't have  correct format"</span>
  }
}</code>
```
✅ Normal color scheme props
```html
<code>{
  <span class="hljs-attr">"errors"</span>: {
    <span class="hljs-attr">"name"</span>: <span style="color: rgb(162, 252, 162);">"Name doesn't have the correct format"</span>,
    <span class="hljs-attr">"firstname"</span>: <span style="color: rgb(162, 252, 162);">"Firstname doesn't have the correct format"</span>,
    <span class="hljs-attr">"username"</span>: <span style="color: rgb(162, 252, 162);">"Username doesn't have the correct format"</span>,
    <span class="hljs-attr">"email"</span>: <span style="color: rgb(162, 252, 162);">"Username doesn't have correct  format"</span>,
    <span class="hljs-attr">"password"</span>: <span style="color: rgb(162, 252, 162);">"Password doesn't have ee correct format"</span>,
    <span class="hljs-attr">"question"</span>: <span style="color: rgb(162, 252, 162);">"Question doesn't have  correct format"</span>,
    <span class="hljs-attr">"reponse"</span>: <span style="color: rgb(162, 252, 162);">"Reponse doesn't have  correct format"</span>
  }
}</code>
```

We can see on the HTML of the broken color scheme, the inline style color being set to 
```css
color: rgb(136, 136, 136); /* or #888 => grey */
```
but on the HTML of the good color scheme, it's set to
```css
color: rgb(162, 252, 162); /* or #a2fca2 => greenish*/
```

But as far as I can say, I'm not an expert in react, as a Vue.js dev, so my job is probably done here ✌️

**Comment by MrStanDu33** [comment]
Hello there !
*~~General Kenobi~~*

Update here, I just checked the progress of this issue, and  it seems that this issue was fixed on develop. I tried to check when it was fixed to undestand what happend.

| Commit                                                                                       | Bug present | Comment                                                                           |
|----------------------------------------------------------------------------------------------|-------------|-----------------------------------------------------------------------------------|
| [#f1ac0f5](https://github.com/Kong/insomnia/commit/f1ac0f5f02295c58fa450137b88a9a8f68f0ce3d) | ❌           | Latest commit on develop                                                          |
| [#f1d6e25](https://github.com/Kong/insomnia/commit/f1d6e25a913a5a3b87e32eebddd6977c4aee8f37) | ❌           | Bug fixed, upgrade to react 17                                                    |
| [#e0b91b5](https://github.com/Kong/insomnia/commit/e0b91b5aedbdd6630a96940e1dfd4a43b7870a32) | ✅           | [Release 2022.2.0](https://github.com/Kong/insomnia/releases/tag/core%402022.2.0) |

So it seems that upgrading to React 17 with [#f1d6e25](https://github.com/Kong/insomnia/commit/f1d6e25a913a5a3b87e32eebddd6977c4aee8f37) fixed the problem in the app. I suppose it was, as I thought, caused by the package [react-syntax-highlighter](https://github.com/react-syntax-highlighter/react-syntax-highlighter).

At this time, [release 2022.2.1](https://github.com/Kong/insomnia/compare/release/2022.2.1) is under preparation, but it does not include commit [#f1d6e25](https://github.com/Kong/insomnia/commit/f1d6e25a913a5a3b87e32eebddd6977c4aee8f37) so it will still include the bug.

I recommand @Kwaay to close this issue as it will be fix in some days, when [release 2022.2.1](https://github.com/Kong/insomnia/compare/release/2022.2.1) will be released if it includes the fix, or the next one.

May ~~the force~~ your day be ~~with you~~ nice.

**Comment by ghost** [comment]
Hello there !
Considering that this problem will be solved with the release [2022.2.1](https://github.com/Kong/insomnia/compare/release/2022.2.1), this issue no longer needs to be opened, thanks @MrStanDu33 for investigating !

The ~~force~~ happiness is strong in you
